server/form-pages/assess/makeADecision/makeADecisionTask/makeADecision.ts

```diff
@@ -65,6 +66,8 @@ export default class MakeADecision implements TasklistPage {
     const errors: TaskListErrors<this> = {}
 
     if (!this.body.decision) errors.decision = 'You must select one option'
+    if (this.body.decision && this.body.decision !== 'accept' && !this.body.rejectReason)
+      errors.rejectReason = 'You must enter reject reason'
```

Comment:
```suggestion
      errors.rejectReason = 'You must enter a reason for rejection'
```

Reply:
Apologies for the late contribution but how about "You must provide the rationale for your decision"?